### PR TITLE
feat: Ability to hide unversioned files in SCM view. fixes #264

### LINKED ISSUE #264: Add config to ignore UNVERSIONED files in UI
- VSCode Version: Latest
- OS Version: All supporting OS
- Extension Version: Latest
- System Language: Any
- SVN Version: Any

## Feature Request
The number of UNVERSIONED files in our code base is huge and the UI is getting cluttered. I was currently making changes to the installed extension code, which gets wiped every time an update was performed.

My Request is to add a new boolean to settings.json (may be "svn.showUnversioned") should be able to hide the unversioned files from the UI.

### PR DESCRIPTION
**Pull request by JohnstonCode**

### CODE REVIEW

File: src/repository.ts [hunk 1]
```diff
@@ -439,6 +443,10 @@ export class Repository {
           /(.+?)\.(mine|working|merge-\w+\.r\d+|r\d+)$/
         );
 
+        if (hideUnversioned) {
```
Comment (edgardmessias) [inline src/repository.ts]: For performance reason, move to before of `const matches = status.path.match(`

**Comment by JohnstonCode** [comment]
@edgardmessias made that change

**Comment by edgardmessias** [comment]
Greate, now it's ok for me

**Pull request merged**